src/schema.ts

```diff
@@ -340,7 +379,7 @@ UserSchema.virtual("userIsActive").get(async function (this: typeof UserSchema)
     accounts: this.accountPath,
   })
 
-  return volume?.outgoingSats > 1000 || volume?.incomingSats > 1000
+  return volume.outgoingSats > 1000 || volume.incomingSats > 1000
```

Comment:
those magic strings should be at minimum set as a `const`. but even better could be set in our config yaml file

Reply:
Fixed